apps/mobile/src/services/notifications/FCMService.ts

```diff
@@ -1,7 +1,9 @@
+//@ts-ignore
+globalThis.RNFB_SILENCE_MODULAR_DEPRECATION_WARNINGS = true
```

Comment:
I think we better move this to the entry point of the application. It's a bit weird to have this global var first in the FCMService. It seems to be a global thing that react-native-firebase "needs".